EXAMPLES.md

```diff
@@ -2,343 +2,389 @@
 
 ## Tables of contents
 
-1. [Entity creation and deletion](#entity-creation-and-deletion)
-    1. [create-account](#create-account)
-    2. [delete-account](#delete-account)
-    3. [create-user](#create-user)
-    4. [delete-user](#delete-user)
-    5. [create-access-key](#create-access-key)
-        1. [for an account](#for-an-account)
-        2. [for a user](#for-a-user)
-    6. [delete-access-key](#delete-access-key)
-2. [Entity listing](#entity-listing)
-    1. [list-accounts](#list-accounts)
-    2. [list-account-users](#list-account-users)
-    3. [list-access-keys](#list-access-keys)
+1. [Cli](#cli)
+    1. [Https](#cli-https)
+    2. [Create account](#cli-create-account)
+    3. [Generate account access key](#cli-generate-account-access-key)
+    4. [List accounts](#cli-list-accounts)
+    5. [Delete account](#cli-delete-account)
+2. [Api](#api)
+    1. [Start](#api-start)
+    2. [Create account](#api-create-account)
+    3. [Generate account access key](#api-generate-account-access-key)
+    4. [List accounts](#api-list-accounts)
+    5. [Delete account](#api-delete-account)
+    6. [Authenticate V2](#api-authenticate-v2)
+    7. [Authenticate V4](#api-authenticate-v4)
+    8. [Get email addresses](#api-get-email-addresses)
+    9. [Get canonical ids](#api-get-canonical-ids)
+3. [Step to set up an account](#step-to-set-up-an-account)
+
+## Cli
+
+This section provides examples on how to use the Vault cli to manage your accounts
+
+The Vault server is assumed to be running at 127.0.0.1:8600 in the following
+examples.
+
+### Https
+
+To enable https for vaultclient cli, you have to pass the option ```--https```
+to vaultclient cli. In the case of self-signed certificates, you also need to
+use the option ```--cafile <certificate-authority-path>```. Another option is
+using the option ```--noCaVerification``` to disable ssl certificate validation.
+
+### Create account
 
-## Entity creation and deletion
-
-This page provides quick example one-liners on how to create and delete entities
-by using the different routes supported by vaultclient and Vault.
-
-See [Vault's design document]
-(https://github.com/scality/Vault/blob/master/Design.md) for a recap
-on how entities are related in our supported IAM model.
-
-The Vault server is assumed to be running at 127.0.0.1:8500 in the following
-examples. Keep in mind that the '--host' option is always mandatory, indicating
-either Vault Server's IP or Fully Qualified Domain Name.
-
-Vaultclient uses HTTP by default, and so do the following examples. Use the
-option '--https' in every command to force the use of HTTPS. To add your own
-certificate authority, also use the option '--cafile /some/path'.
+```sh
+vautclient create-account --name test --email test@test.com
```

Comment:
I think the examples would be more vivid if we use some fictional names here instead of test. Lets call it
`vaultclient create-account --name john --email john@acme.com`